docs/book/src/introduction/sway_quickstart.md

```diff
@@ -173,10 +173,15 @@ This starts a Fuel node with a volatile database that will be cleared when shut
 
 To deploy `counter_contract` on your local Fuel node, open a new terminal tab and run the following command from the root of the `wallet_contract` directory:
 
+> **Note**
+> You will require the plugin [forc-wallet](https://github.com/FuelLabs/forc-wallet) in order to sign your transactions.
+
 ```sh
-forc deploy --unsigned
+forc deploy --url localhost:4000 --unsigned
 ```
 
+where `--url` parameter is the URL where your local node is hosted.
+
```

Comment:
^

Reply:
@leirbag95, since we removed the `--url` from line 180, this seems to have no context. Other than that I will happily approve. We can rewrite it to mention `--url` can be used to specify the local node or we can remove the line all together.